pom.xml

```diff
@@ -294,6 +294,32 @@
                 </repository>
             </distributionManagement>
         </profile>
+        <profile>
+            <id>coverage</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.jacoco</groupId>
+                        <artifactId>jacoco-maven-plugin</artifactId>
+                        <version>0.8.5</version>
```

Comment:
Perhaps a property for JaCoCo version would be better

Reply:
property is now used